### ISSUE TITLE
[req]:hdgalaktik

### ISSUE THREAD

**Issue by berat1397** [Jackett/Jackett#13186]
### Is there already a request for your tracker?

- [X] I have checked older issues, open and closed

### Type

Semi-Private

### Tracker details

```markdown
- Name: HDGalaKtik
- URL: https://hdgalaktik.com/
- Description: HDGalaKtik is a RUSSIAN Semi-Private tracker for TV / MOVIES / GENERAL
- Minimum Seeding Time and Minimum Ratio (if Private): 'optimal ratio' is mentioned as 1:1, not sure if it's actually enforced, couldn't see MST
- Additional Comments: Big russian torrent websites.
```

**Comment by garfield69** [comment]
v0.20.989

**Comment by ilike2burnthing** [comment]
When failing to add the indexer the HTML from the enhanced log shows I'm being presented with their captcha:
![image](https://user-images.githubusercontent.com/59480337/166127359-c7c7a2ad-ebaa-4f26-a7c8-5d737faaa789.png)

Can you confirm this or am I doing something wrong on my end (e.g. some tracker account setting)?

**Comment by garfield69** [comment]
yes, if you fail the login then the next time you get a challenge. 
also, when I've used the indexer for a search and click on the details of a torrent, I often get a browser blocked for 10 minutes due to the Jackett session being considered 'still active' !?!
PITA

**Comment by ilike2burnthing** [comment]
Might it be worth switching to cookie login then so people can still use the website? Or is the TTL annoyingly short (I haven't checked)?

**Comment by garfield69** [comment]
I'll set up a test and see how long the cookie lasts.

**Comment by garfield69** [comment]
LGTM so far. I can browse the site and use the indexer without conflicts.
The PHPSESSID is set to expire when the browsing session ends and the other 5 keys not until jan 2038
I'm going to push it now.

**Comment by garfield69** [comment]
v0.20.993

**Comment by garfield69** [comment]
https://www.trackerpmr.com/ ERR_CONNECTION_TIMED_OUT

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
site is broken, with error blocks containing `Warning: include_once(/var/www/www-root/data/www/trackerpmr.com//languages/lang_english/lang_main.php): failed to open stream: No such file or directory in /var/www/www-root/data/www/trackerpmr.com/include/functions.php on line 336` being inserted in results, downloads, all over the place.
also, the categories have gone with the `a[href^="browse.php?cat="]` missing.
By forcing cat=4 (other) you can progress to see results, but starting a DL causes a bencode failure as that error block above contaminates the .torrent file.

**Comment by garfield69** [comment]
ilike2burnthing Can you confirm that it is not just me that is getting these warning html blocks embedded everyplace about the English language function not found? Does your indexer also fail to fetch a torrent with a bencode failure?

**Comment by ilike2burnthing** [comment]
Yep, same here: site and torrents broken, categories missing.

**Comment by ilike2burnthing** [comment]
Site is no longer throwing error blocks everywhere, and torrents work, but categories are still missing.

Going to https://www.trackerpmr.com/my.php, and then clicking on the `Другое` section, shows that many of the settings titles are missing. Above the categories at the bottom there is a 4x3 set of checkboxes, the top left of which is `cat` in the HTML. Unfortunately disabling it, enabling it again, disabling everything, checking all the categories, or any combination of the above, does not enable categories for results. However, changing these does otherwise affect the layout, so I think categories are supposed to be there.

Rather than wait to see if categories return, I'm going to keep `a[href^="browse.php?cat="]` as the selector, and use `4` as the default.

**Comment by garfield69** [comment]
v0.21.2476

**Comment by garfield69** [comment]
https://kinoradiomagia.tv/browse.php 503 Service Temporarily Unavailable
Attention! Technical work is underway on the site! # 8.6

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://www.trackerpmr.com/ > 503 Service Temporarily Unavailable

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
https://kinoradiomagia.tv/ > timeout
https://freetmd.com/ > cloudflare  Connection timed out Error code 522 
https://www.trackerpmr.com/ > timeout (on browser albeit jackett indexer seems to be working reliably %-/ )

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
site changed browse.php layout from table/tr/td to div/div/div, indexer update in progress

**Comment by garfield69** [comment]
site changed browse.php layout back to table/tr/td, but its slightly different from the original, indexer update in progress

**Comment by garfield69** [comment]
Oh F**k this, they have changed the layout again just as I finished updating.
I'm going to leave this for later, I don't have the time to waste to chase this crap this morning.

**Comment by garfield69** [comment]
took a look just now, that browse page does not have categories, and if you search using the cats from the pulldown you don't get anything. I suspect the site is still a work in progress so I shall not be updating the indexer at this time.

**Comment by garfield69** [comment]
Typical. Site has changed the layout AGAIN!

**Comment by garfield69** [comment]
still cannot search by category but at least the cats are back in results.

**Comment by garfield69** [comment]
crappy website. I'm nearly done re-coding indexer and the site is not returning any results, internal server error, jeesh
i'll take another look alter today

**Comment by garfield69** [comment]
https://www.trackerpmr.com/browse.php?cat[]=0&shw_incl_cats=0&incldead=0&sort=4&type=desc > InternalServerError

**Comment by garfield69** [comment]
Searching for a title works, but a keywordless search does not.
and searching with 2025, or any of the years does not work either.

There is a latest torrents page, which the site generates with https://www.trackerpmr.com/browse.php?date=2025-11-30
but cardigann does not have any way to build that date, unless we add .Today.Month and .Today.Day 
or write the indexer in C# which I am not interested in doing.

**Comment by ilike2burnthing** [comment]
`2025` is currently working for me in browser and Jackett. No idea if they're just doing even more changes 🙄

**Comment by garfield69** [comment]
zero results, looks like the site has changed selectors again

**Comment by garfield69** [comment]
as is typical of this site, the moment I complete my layout change the site starts returning zero results with another layout change Grrrr. 
placing it on hold for now, I'll try again later.

**Comment by garfield69** [comment]
the site no longer returns a default list when using the browse.php page, and &search=2026 returns zero results as does any 4 digit date, but a specific search works such as search=noches
if we cannot get a keywordless search going then I see little point proceeding.
<details>
  <summary>indexer</summary>

```yaml
---
id: hdgalaktik
name: HDGalaKtik
description: "HDGalaKtik is a RUSSIAN Semi-Private tracker for MOVIES / TV / GENERAL"
language: ru-RU
type: semi-private
encoding: UTF-8
links:
  - https://www.trackerpmr.com/
  - https://freetmd.com/
  - https://kinoradiomagia.tv/
legacylinks:
  - https://hdgalaktik.com/

caps:
  categorymappings:
    - {id: 13, cat: Movies, desc: "Фильмы"}
    - {id: 11, cat: TV, desc: "Сериалы"}
    - {id: 6, cat: TV, desc: "Мультфильмы"}
    - {id: 10, cat: Audio, desc: "Музыка"}
    - {id: 26, cat: PC, desc: "Программы"}
    - {id: 5, cat: Console, desc: "Игры"}
    - {id: 25, cat: Other, desc: "Картинки"}
    - {id: 12, cat: TV/Anime, desc: "Аниме"}
    - {id: 16, cat: Books, desc: "Книги"}
    - {id: 18, cat: Audio/Video, desc: "Клипы / Ролики"}
    - {id: 22, cat: TV, desc: "ТВ / Передачи"}
    - {id: 27, cat: PC/Mobile-Other, desc: "Игры - Мобила / КПК"}
    - {id: 1, cat: PC/ISO, desc: "Образы"}
    - {id: 4, cat: Other, desc: "Другое"}

  modes:
    search: [q]
    tv-search: [q, season, ep]
    movie-search: [q]
    music-search: [q]
    book-search: [q]

settings:
  - name: cookie
    type: text
    label: Cookie
  - name: info_cookie
    type: info_cookie
  - name: stripcyrillic
    type: checkbox
    label: Strip Cyrillic Letters
    default: false
  - name: addrussiantotitle
    type: checkbox
    label: Add RUS to end of all titles to improve language detection by Sonarr and Radarr. Will cause English-only results to be misidentified.
    default: false
  - name: freeleech
    type: checkbox
    label: Search freeleech only
    default: false
  - name: sort
    type: select
    label: Sort requested from site
    default: 4
    options:
      4: created
      1: title
      5: size
      7: seeders
  - name: type
    type: select
    label: Order requested from site
    default: desc
    options:
      desc: desc
      asc: asc
  - name: info_tpp
    type: info
    label: Results Per Page
    default: For best results, change the <b>Torrents per page:</b> setting to <b>100</b> on your account profile.

login:
  # using the cookie method because the site expects a single session only, so using browser and Jackett can cause a lockout
  method: cookie
  inputs:
    cookie: "{{ .Config.cookie }}"
  test:
    path: /
    selector: a[href="mybonus.php"]

search:
  # https://freetmd.com/browse.php?cat[]=13&cat[]=11&shw_incl_cats=0&incldead=0#results
  paths:
    - path: browse.php
  inputs:
    # 0 default, 1 active, 2 dead, 3 gold, 4 sticky, lots of others
    incldead: "{{ if .Config.freeleech }}3{{ else }}0{{ end }}"
    search: "{{ if .Keywords }}{{ .Keywords }}{{ else }}{{ .Today.Year }}{{ end }}"
    sort: "{{ .Config.sort }}"
    type: "{{ .Config.type }}"

  keywordsfilters:
    - name: re_replace # S01 to сезон 1
      args: ["(?i)\\bS0*(\\d+)\\b", "сезон $1"]
    - name: re_replace # E02 to сери 1
      args: ["(?i)\\bE0*(\\d+)\\b", "сери $1"]
    - name: re_replace # S01E02 to сезон 1 сери 2
      args: ["(?i)\\bS0*(\\d+)E0*(\\d+)\\b", "сезон $1 сери $2"]
    - name: re_replace # replace special characters with "%" (wildcard)
      args: ["[^a-zA-Z0-9]+", "%"]

  rows:
    selector: div.vb-item

  fields:
    category:
      selector: a[href^="browse.php?cat="]
      attribute: href
      filters:
        - name: querystring
          args: cat
    title:
      selector: h3 > a
      attribute: title
      filters:
        # normalize to SXXEYY format
        - name: re_replace
          args: ["(?i)[CС]езоны?[\\s:]*(\\d+(?:-\\d+)?).+?(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))[\\s:]*(\\d+(?:-\\d+)?)\\s*из\\s*(\\w?)", "S$1E$2 of $3"]
        - name: re_replace
          args: ["(?i)(\\d+(?:-\\d+)?)\\s*[CС]езоны?.+?(\\d+(?:-\\d+)?)\\s*из\\s*(\\w?)(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))?", "S$1E$2 of $3"]
        - name: re_replace
          args: ["(?i)(\\d+(?:-\\d+)?)\\s*[CС]езоны?.+?(\\d+(?:-\\d+)?)\\s*(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))\\s+из\\s*(\\w?)", "S$1E$2 of $3"]
        - name: re_replace
          args: ["(?i)[CС]езоны?[\\s:]*(\\d+(?:-\\d+)?).+?(\\d+(?:-\\d+)?)\\s*из\\s*(\\w?)(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))?", "S$1E$2 of $3"]
        - name: re_replace
          args: ["(?i)[CС]езоны?[\\s:]*(\\d+(?:-\\d+)?).+?(\\d+(?:-\\d+)?)\\s*(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))\\s+из\\s*(\\w?)", "S$1E$2 of $3"]
        - name: re_replace
          args: ["(?i)[CС]езоны?[\\s:]*(\\d+(?:-\\d+)?).+?(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))[\\s:]*(\\d+(?:-\\d+)?)", "S$1E$2"]
        - name: re_replace
          args: ["(?i)(\\d+(?:-\\d+)?)\\s*[CС]езоны?.+?(\\d+(?:-\\d+)?)(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))", "S$1E$2"]
        - name: re_replace
          args: ["(?i)[CС]езоны?[\\s:]*(\\d+(?:-\\d+)?).+?(\\d+(?:-\\d+)?)(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))", "S$1E$2"]
        - name: re_replace
          args: ["(?i)[CС]езоны?[\\s:]*(\\d+(?:-\\d+)?)", "S$1"]
        - name: re_replace
          args: ["(?i)(\\d+(?:-\\d+)?)\\s+[CС]езоны?", "S$1"]
        - name: re_replace
          args: ["(?i)(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))[\\s:]*(\\d+(?:-\\d+)?)\\s*из\\s*(\\w?)", "E$1 of $2"]
        - name: re_replace
          args: ["(?i)(\\d+(?:-\\d+)?)\\s*из\\s*(\\w?)(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))", "E$1 of $2"]
        - name: re_replace
          args: ["(?i)(\\d+(?:-\\d+)?)\\s+(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))\\s+из\\s*(\\w?)", "E$1 of $2"]
        - name: re_replace
          args: ["(?i)(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))[\\s:]*(\\d+(?:-\\d+)?)", "E$1"]
        - name: re_replace
          args: ["(?i)(\\d+(?:-\\d+)?)\\s+(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))", "E$1"]
        - name: re_replace
          args: ["(?i)\\bFiles-х\\b", "Files-x"]
        - name: re_replace
          args: ["(?i)\\sот\\s([\\w\\p{P}\\p{S}]+)$", "-$1"]
        - name: re_replace
          args: ["\\s\\|\\s(\\w{4,})$", "-$1"]
        - name: re_replace
          args: ["(\\([\\p{IsCyrillic}\\W]+\\))|(^[\\p{IsCyrillic}\\W\\d]+\\/ )|([\\p{IsCyrillic} \\-]+,+)|([\\p{IsCyrillic}]+)", "{{ if .Config.stripcyrillic }}{{ else }}$1$2$3$4{{ end }}"]
        - name: re_replace
          args: ["(?i)\\bHDTV[-\\s]?Rip\\b", "HDTV"]
        - name: re_replace
          args: ["(?i)\\bSAT[-\\s]?Rip\\b", "HDTV"]
        - name: re_replace
          args: ["(?i)\\bWEB[-\\s]?DL[-\\s]?Rip\\b", "WEB-DL"]
        - name: re_replace
          args: ["(?i)\\bWEB\\sRip\\b", "WEBRip"]
        - name: re_replace
          args: ["(?i)\\bWEB\\sDL\\b", "WEB-DL"]
        - name: re_replace
          args: ["[\\[\\(\\{<«][\\s\\W]*[\\]\\)\\}>»]", ""]
        - name: re_replace
          args: ["^[\\s&,\\.!\\?\\+\\-_\\|\\/':]+", ""]
        - name: re_replace
          args: ["(?i)^\\(\\s*([SE]\\d+.*?)\\s*\\)[\\s\\/\\|]*(.+)", "$2 $1"]
        - name: append
          args: "{{ if .Config.addrussiantotitle }} RUS{{ else }}{{ end }}"
    details:
      selector: h3 > a
      attribute: href
    download:
      selector: a[href^="download.php?id="]
      attribute: href
      optional: true # for users that do not have DL access
    poster:
      selector: picture > img
      attribute: src
      filters:
        - name: replace
          args: ["/images/default_torrent.png", ""]
    size:
      selector: div.vb-stats > div:nth-child(3) > span:nth-child(2)
    seeders:
      selector: div.vb-stats > div:nth-child(5) > span:nth-child(2)
    leechers:
      selector: div.vb-stats > div:nth-child(6) > span:nth-child(2)
    date:
      selector: div.vb-meta > span:nth-child(3)
      filters:
        - name: timeago
    grabs:
      selector: div.vb-meta > span:nth-child(5)
    files:
      selector: div.vb-meta > span:nth-child(7)
    downloadvolumefactor:
      case:
        span.vb-badge-free: 0
        "*": 1
    uploadvolumefactor:
      text: 1
    minimumratio:
      text: 1.0
# engine n/a
```

</details>

**Comment by ilike2burnthing** [comment]
Not sure if there's been another update or if my layout is different, but I'm getting a default 'latest' list on `browse.php`.

If that's working for you now as well, then `search` can be:

```yml
    search: "{{ if .Keywords }}{{ .Keywords }}{{ else }}{{ end }}"
```

Also ran into an issue with some results having a group listed as well as the uploader, so `date`, `grabs`, and `files` needed updated:

```yml
    date:
      selector: div.vb-meta > span:nth-last-child(5)
      filters:
        - name: timeago
    grabs:
      selector: div.vb-meta > span:nth-last-child(3)
    files:
      selector: div.vb-meta > span:nth-last-child(1)
```

**Comment by garfield69** [comment]
I've turned on developer tools for my browser and when I access the browser.php I see the return code 500 internal server error, but the page is still being generated with the search box, but with no results.
So for some reason the browse page is broken for me.
So I'm bailing out of this one and inviting you to complete it as you see fit.

**Comment by garfield69** [comment]
found the reason for the constant http 500 when browsing torrents on the site, it was my language settings on my sites profile which was set to English. I changed it to Russian and now I get results.
however on the indexer I get zero. is the indexer working for you:?

**Comment by garfield69** [comment]
ok interesting. i reverted the update to the indexer and that is now working for me.
I cant find a setting that changes the browse.php layout on the web site.
so is this just me?

**Comment by garfield69** [comment]
v0.24.2014

**Comment by garfield69** [comment]
reverted v0.24.2021

**Comment by garfield69** [comment]
https://kinoradiomagia.tv/ >  TCP Connection Failed (Timed out)
https://freetmd.com/ > CLoudflare Connection timed out Error code 522
https://www.trackerpmr.com/ > TCP Connection Failed (Timed out)

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
hdgalaktik have changed layout again.

**Comment by garfield69** [comment]
I give up, I can get it to pick up rows %-/
<details>
  <summary>hdgalaktic</summary>

```yaml
---
id: hdgalaktik
name: HDGalaKtik
description: "HDGalaKtik is a RUSSIAN Semi-Private tracker for MOVIES / TV / GENERAL"
language: ru-RU
type: semi-private
encoding: UTF-8
links:
  - https://www.trackerpmr.com/
  - https://freetmd.com/
  - https://kinoradiomagia.tv/
legacylinks:
  - https://hdgalaktik.com/

caps:
  categorymappings:
    - {id: 13, cat: Movies, desc: "Фильмы"}
    - {id: 11, cat: TV, desc: "Сериалы"}
    - {id: 6, cat: TV, desc: "Мультфильмы"}
    - {id: 10, cat: Audio, desc: "Музыка"}
    - {id: 26, cat: PC, desc: "Программы"}
    - {id: 5, cat: Console, desc: "Игры"}
    - {id: 25, cat: Other, desc: "Картинки"}
    - {id: 12, cat: TV/Anime, desc: "Аниме"}
    - {id: 16, cat: Books, desc: "Книги"}
    - {id: 18, cat: Audio/Video, desc: "Клипы / Ролики"}
    - {id: 22, cat: TV, desc: "ТВ / Передачи"}
    - {id: 27, cat: PC/Mobile-Other, desc: "Игры - Мобила / КПК"}
    - {id: 1, cat: PC/ISO, desc: "Образы"}
    - {id: 4, cat: Other, desc: "Другое"}

  modes:
    search: [q]
    tv-search: [q, season, ep]
    movie-search: [q]
    music-search: [q]
    book-search: [q]

settings:
  - name: info_view
    type: info
    label: Torrent page view
    default: Before you fetch the cookie from your browser visit the <b>Торренты (Torrent)</b> page and select the <b>Таблица (Table)</b> view. Selecting either the <i>Блоки (Grid)</i> or <i>Список (List)</i> views will return zero results.
  - name: cookie
    type: text
    label: Cookie
  - name: info_cookie
    type: info_cookie
  - name: stripcyrillic
    type: checkbox
    label: Strip Cyrillic Letters
    default: false
  - name: addrussiantotitle
    type: checkbox
    label: Add RUS to end of all titles to improve language detection by Sonarr and Radarr. Will cause English-only results to be misidentified.
    default: false
  - name: freeleech
    type: checkbox
    label: Search freeleech only
    default: false
  - name: sort
    type: select
    label: Sort requested from site
    default: 4
    options:
      4: created
      1: title
      5: size
      7: seeders
  - name: type
    type: select
    label: Order requested from site
    default: desc
    options:
      desc: desc
      asc: asc
  - name: info_tpp
    type: info
    label: Results Per Page
    default: For best results, change the <b>Torrents per page:</b> setting to <b>100</b> on your account profile.

login:
  # using the cookie method because the site expects a single session only, so using browser and Jackett can cause a lockout
  method: cookie
  inputs:
    cookie: "{{ .Config.cookie }}"
  test:
    path: /
    selector: a[href="mybonus.php"]

search:
  # https://www.trackerpmr.com/browse.php?cat[]=0&shw_incl_cats=0&incldead=0&search=the+fix&tag=#results
  paths:
    - path: browse.php
  inputs:
    # 0 default, 1 active, 2 dead, 3 gold, 4 sticky, lots of others
    incldead: "{{ if .Config.freeleech }}3{{ else }}0{{ end }}"
    search: "{{ .Keywords }}"
    sort: "{{ .Config.sort }}"
    type: "{{ .Config.type }}"

  keywordsfilters:
    - name: re_replace # S01 to сезон 1
      args: ["(?i)\\bS0*(\\d+)\\b", "сезон $1"]
    - name: re_replace # E02 to сери 1
      args: ["(?i)\\bE0*(\\d+)\\b", "сери $1"]
    - name: re_replace # S01E02 to сезон 1 сери 2
      args: ["(?i)\\bS0*(\\d+)E0*(\\d+)\\b", "сезон $1 сери $2"]
    - name: re_replace # replace special characters with "%" (wildcard)
      args: ["[^a-zA-Z0-9]+", "%"]

  rows:
    selector: tr[class*="torcont"]
    after: 1

  fields:
    category:
      selector: a[href^="browse.php?cat="]
      attribute: href
      filters:
        - name: querystring
          args: cat
    title:
      selector: a.tv-card-title
      attribute: title
      filters:
        # normalize to SXXEYY format
        - name: re_replace
          args: ["(?i)[CС]езоны?[\\s:]*(\\d+(?:-\\d+)?).+?(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))[\\s:]*(\\d+(?:-\\d+)?)\\s*из\\s*(\\w?)", "S$1E$2 of $3"]
        - name: re_replace
          args: ["(?i)(\\d+(?:-\\d+)?)\\s*[CС]езоны?.+?(\\d+(?:-\\d+)?)\\s*из\\s*(\\w?)(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))?", "S$1E$2 of $3"]
        - name: re_replace
          args: ["(?i)(\\d+(?:-\\d+)?)\\s*[CС]езоны?.+?(\\d+(?:-\\d+)?)\\s*(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))\\s+из\\s*(\\w?)", "S$1E$2 of $3"]
        - name: re_replace
          args: ["(?i)[CС]езоны?[\\s:]*(\\d+(?:-\\d+)?).+?(\\d+(?:-\\d+)?)\\s*из\\s*(\\w?)(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))?", "S$1E$2 of $3"]
        - name: re_replace
          args: ["(?i)[CС]езоны?[\\s:]*(\\d+(?:-\\d+)?).+?(\\d+(?:-\\d+)?)\\s*(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))\\s+из\\s*(\\w?)", "S$1E$2 of $3"]
        - name: re_replace
          args: ["(?i)[CС]езоны?[\\s:]*(\\d+(?:-\\d+)?).+?(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))[\\s:]*(\\d+(?:-\\d+)?)", "S$1E$2"]
        - name: re_replace
          args: ["(?i)(\\d+(?:-\\d+)?)\\s*[CС]езоны?.+?(\\d+(?:-\\d+)?)(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))", "S$1E$2"]
        - name: re_replace
          args: ["(?i)[CС]езоны?[\\s:]*(\\d+(?:-\\d+)?).+?(\\d+(?:-\\d+)?)(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))", "S$1E$2"]
        - name: re_replace
          args: ["(?i)[CС]езоны?[\\s:]*(\\d+(?:-\\d+)?)", "S$1"]
        - name: re_replace
          args: ["(?i)(\\d+(?:-\\d+)?)\\s+[CС]езоны?", "S$1"]
        - name: re_replace
          args: ["(?i)(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))[\\s:]*(\\d+(?:-\\d+)?)\\s*из\\s*(\\w?)", "E$1 of $2"]
        - name: re_replace
          args: ["(?i)(\\d+(?:-\\d+)?)\\s*из\\s*(\\w?)(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))", "E$1 of $2"]
        - name: re_replace
          args: ["(?i)(\\d+(?:-\\d+)?)\\s+(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))\\s+из\\s*(\\w?)", "E$1 of $2"]
        - name: re_replace
          args: ["(?i)(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))[\\s:]*(\\d+(?:-\\d+)?)", "E$1"]
        - name: re_replace
          args: ["(?i)(\\d+(?:-\\d+)?)\\s+(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))", "E$1"]
        - name: re_replace
          args: ["(?i)\\bFiles-х\\b", "Files-x"]
        - name: re_replace
          args: ["(?i)\\sот\\s([\\w\\p{P}\\p{S}]+)$", "-$1"]
        - name: re_replace
          args: ["\\s\\|\\s(\\w{4,})$", "-$1"]
        - name: re_replace
          args: ["(\\([\\p{IsCyrillic}\\W]+\\))|(^[\\p{IsCyrillic}\\W\\d]+\\/ )|([\\p{IsCyrillic} \\-]+,+)|([\\p{IsCyrillic}]+)", "{{ if .Config.stripcyrillic }}{{ else }}$1$2$3$4{{ end }}"]
        - name: re_replace
          args: ["(?i)\\bHDTV[-\\s]?Rip\\b", "HDTV"]
        - name: re_replace
          args: ["(?i)\\bSAT[-\\s]?Rip\\b", "HDTV"]
        - name: re_replace
          args: ["(?i)\\bWEB[-\\s]?DL[-\\s]?Rip\\b", "WEB-DL"]
        - name: re_replace
          args: ["(?i)\\bWEB\\sRip\\b", "WEBRip"]
        - name: re_replace
          args: ["(?i)\\bWEB\\sDL\\b", "WEB-DL"]
        - name: re_replace
          args: ["[\\[\\(\\{<«][\\s\\W]*[\\]\\)\\}>»]", ""]
        - name: re_replace
          args: ["^[\\s&,\\.!\\?\\+\\-_\\|\\/':]+", ""]
        - name: re_replace
          args: ["(?i)^\\(\\s*([SE]\\d+.*?)\\s*\\)[\\s\\/\\|]*(.+)", "$2 $1"]
        - name: append
          args: "{{ if .Config.addrussiantotitle }} RUS{{ else }}{{ end }}"
    details:
      selector: a[href^="details.php?id="]
      attribute: href
    download:
      selector: a[href^="download.php?id="]
      attribute: href
      optional: true # for users that do not have DL access
    poster:
      selector: img.tv-avatars
      attribute: src
    size:
      selector: a.tv-dl-btn
    seeders:
      selector: span.up
    leechers:
      selector: span.down
    grabs:
      selector: td:nth-last-child(7)
    date:
      selector: a[href^="browse.php?date="]
      attribute: href
      filters:
        - name: querystring
          args: date
        - name: dateparse
          args: "yyyy-MM-dd"
    downloadvolumefactor:
      case:
        span.tv-badge-free: 0
        "*": 1
    uploadvolumefactor:
      text: 1
    minimumratio:
      text: 1.0
    description:
      selector: a.tr-tag
# Tesla v.Core (8.7.18)
```
</details>

**Comment by garfield69** [comment]
I've managed to get the indexer working with grid, so if you dont have any joy with table then I'll push the grid update instead.

**Comment by ilike2burnthing** [comment]
This is what I got, if you want to compare:

<details>
<summary>hdgalaktik</summary>

```yml
---
id: hdgalaktik
name: HDGalaKtik
description: "HDGalaKtik is a RUSSIAN Semi-Private tracker for MOVIES / TV / GENERAL"
language: ru-RU
type: semi-private
encoding: UTF-8
links:
  - https://www.trackerpmr.com/
  - https://freetmd.com/
  - https://kinoradiomagia.tv/
legacylinks:
  - https://hdgalaktik.com/

caps:
  categorymappings:
    - {id: 13, cat: Movies, desc: "Фильмы"}
    - {id: 11, cat: TV, desc: "Сериалы"}
    - {id: 6, cat: TV, desc: "Мультфильмы"}
    - {id: 10, cat: Audio, desc: "Музыка"}
    - {id: 26, cat: PC, desc: "Программы"}
    - {id: 5, cat: Console, desc: "Игры"}
    - {id: 25, cat: Other, desc: "Картинки"}
    - {id: 12, cat: TV/Anime, desc: "Аниме"}
    - {id: 16, cat: Books, desc: "Книги"}
    - {id: 18, cat: Audio/Video, desc: "Клипы / Ролики"}
    - {id: 22, cat: TV, desc: "ТВ / Передачи"}
    - {id: 27, cat: PC/Mobile-Other, desc: "Игры - Мобила / КПК"}
    - {id: 1, cat: PC/ISO, desc: "Образы"}
    - {id: 4, cat: Other, desc: "Другое"}

  modes:
    search: [q]
    tv-search: [q, season, ep]
    movie-search: [q]
    music-search: [q]
    book-search: [q]

settings:
  - name: cookie
    type: text
    label: Cookie
  - name: info_cookie
    type: info_cookie
  - name: stripcyrillic
    type: checkbox
    label: Strip Cyrillic Letters
    default: false
  - name: addrussiantotitle
    type: checkbox
    label: Add RUS to end of all titles to improve language detection by Sonarr and Radarr. Will cause English-only results to be misidentified.
    default: false
  - name: freeleech
    type: checkbox
    label: Search freeleech only
    default: false
  - name: sort
    type: select
    label: Sort requested from site
    default: 4
    options:
      4: created
      1: title
      5: size
      7: seeders
  - name: type
    type: select
    label: Order requested from site
    default: desc
    options:
      desc: desc
      asc: asc
  - name: info_tpp
    type: info
    label: Results Per Page
    default: For best results, change the <b>Torrents per page:</b> setting to <b>100</b> on your account profile.

login:
  # using the cookie method because the site expects a single session only, so using browser and Jackett can cause a lockout
  method: cookie
  inputs:
    cookie: "{{ .Config.cookie }}"
  test:
    path: /
    selector: a[href="mybonus.php"]

search:
  # https://www.trackerpmr.com/browse.php?cat[]=0&shw_incl_cats=0&incldead=0&search=the+fix&tag=#results
  paths:
    - path: browse.php
  inputs:
    # 0 default, 1 active, 2 dead, 3 gold, 4 sticky, lots of others
    incldead: "{{ if .Config.freeleech }}3{{ else }}0{{ end }}"
    search: "{{ .Keywords }}"
    sort: "{{ .Config.sort }}"
    type: "{{ .Config.type }}"

  keywordsfilters:
    - name: re_replace # S01 to сезон 1
      args: ["(?i)\\bS0*(\\d+)\\b", "сезон $1"]
    - name: re_replace # E02 to сери 1
      args: ["(?i)\\bE0*(\\d+)\\b", "сери $1"]
    - name: re_replace # S01E02 to сезон 1 сери 2
      args: ["(?i)\\bS0*(\\d+)E0*(\\d+)\\b", "сезон $1 сери $2"]
    - name: re_replace # replace special characters with "%" (wildcard)
      args: ["[^a-zA-Z0-9]+", "%"]

  headers:
    Cookie: ["torrentviewblock=1"]

  rows:
    selector: div.tv-card

  fields:
    categorydesc:
      selector: span.tv-card-cat
    title:
      selector: a.tv-card-title
      filters:
        # normalize to SXXEYY format
        - name: re_replace
          args: ["(?i)[CС]езоны?[\\s:]*(\\d+(?:-\\d+)?).+?(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))[\\s:]*(\\d+(?:-\\d+)?)\\s*из\\s*(\\w?)", "S$1E$2 of $3"]
        - name: re_replace
          args: ["(?i)(\\d+(?:-\\d+)?)\\s*[CС]езоны?.+?(\\d+(?:-\\d+)?)\\s*из\\s*(\\w?)(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))?", "S$1E$2 of $3"]
        - name: re_replace
          args: ["(?i)(\\d+(?:-\\d+)?)\\s*[CС]езоны?.+?(\\d+(?:-\\d+)?)\\s*(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))\\s+из\\s*(\\w?)", "S$1E$2 of $3"]
        - name: re_replace
          args: ["(?i)[CС]езоны?[\\s:]*(\\d+(?:-\\d+)?).+?(\\d+(?:-\\d+)?)\\s*из\\s*(\\w?)(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))?", "S$1E$2 of $3"]
        - name: re_replace
          args: ["(?i)[CС]езоны?[\\s:]*(\\d+(?:-\\d+)?).+?(\\d+(?:-\\d+)?)\\s*(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))\\s+из\\s*(\\w?)", "S$1E$2 of $3"]
        - name: re_replace
          args: ["(?i)[CС]езоны?[\\s:]*(\\d+(?:-\\d+)?).+?(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))[\\s:]*(\\d+(?:-\\d+)?)", "S$1E$2"]
        - name: re_replace
          args: ["(?i)(\\d+(?:-\\d+)?)\\s*[CС]езоны?.+?(\\d+(?:-\\d+)?)(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))", "S$1E$2"]
        - name: re_replace
          args: ["(?i)[CС]езоны?[\\s:]*(\\d+(?:-\\d+)?).+?(\\d+(?:-\\d+)?)(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))", "S$1E$2"]
        - name: re_replace
          args: ["(?i)[CС]езоны?[\\s:]*(\\d+(?:-\\d+)?)", "S$1"]
        - name: re_replace
          args: ["(?i)(\\d+(?:-\\d+)?)\\s+[CС]езоны?", "S$1"]
        - name: re_replace
          args: ["(?i)(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))[\\s:]*(\\d+(?:-\\d+)?)\\s*из\\s*(\\w?)", "E$1 of $2"]
        - name: re_replace
          args: ["(?i)(\\d+(?:-\\d+)?)\\s*из\\s*(\\w?)(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))", "E$1 of $2"]
        - name: re_replace
          args: ["(?i)(\\d+(?:-\\d+)?)\\s+(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))\\s+из\\s*(\\w?)", "E$1 of $2"]
        - name: re_replace
          args: ["(?i)(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))[\\s:]*(\\d+(?:-\\d+)?)", "E$1"]
        - name: re_replace
          args: ["(?i)(\\d+(?:-\\d+)?)\\s+(?:\\s*(?:[CС]ери[ияй]|Эпизод|Выпуски?))", "E$1"]
        - name: re_replace
          args: ["(?i)\\bFiles-х\\b", "Files-x"]
        - name: re_replace
          args: ["(?i)\\sот\\s([\\w\\p{P}\\p{S}]+)$", "-$1"]
        - name: re_replace
          args: ["\\s\\|\\s(\\w{4,})$", "-$1"]
        - name: re_replace
          args: ["(\\([\\p{IsCyrillic}\\W]+\\))|(^[\\p{IsCyrillic}\\W\\d]+\\/ )|([\\p{IsCyrillic} \\-]+,+)|([\\p{IsCyrillic}]+)", "{{ if .Config.stripcyrillic }}{{ else }}$1$2$3$4{{ end }}"]
        - name: re_replace
          args: ["(?i)\\bHDTV[-\\s]?Rip\\b", "HDTV"]
        - name: re_replace
          args: ["(?i)\\bSAT[-\\s]?Rip\\b", "HDTV"]
        - name: re_replace
          args: ["(?i)\\bWEB[-\\s]?DL[-\\s]?Rip\\b", "WEB-DL"]
        - name: re_replace
          args: ["(?i)\\bWEB\\sRip\\b", "WEBRip"]
        - name: re_replace
          args: ["(?i)\\bWEB\\sDL\\b", "WEB-DL"]
        - name: re_replace
          args: ["[\\[\\(\\{<«][\\s\\W]*[\\]\\)\\}>»]", ""]
        - name: re_replace
          args: ["^[\\s&,\\.!\\?\\+\\-_\\|\\/':]+", ""]
        - name: re_replace
          args: ["(?i)^\\(\\s*([SE]\\d+.*?)\\s*\\)[\\s\\/\\|]*(.+)", "$2 $1"]
        - name: append
          args: "{{ if .Config.addrussiantotitle }} RUS{{ else }}{{ end }}"
    details:
      selector: a[href^="details.php?id="]
      attribute: href
    download:
      selector: a[href^="download.php?id="]
      attribute: href
      optional: true # for users that do not have DL access
    poster:
      selector: img.tv-avatars
      attribute: src
    size:
      selector: span.tv-dl-btn
    seeders:
      selector: span.up
    leechers:
      selector: span.down
    date:
      text: now
    downloadvolumefactor:
      case:
        span.tv-badge-free: 0
        "*": 1
    uploadvolumefactor:
      text: 1
    minimumratio:
      text: 1.0
    description:
      selector: div.tv-card-tags
# Tesla v.Core (8.7.18)
```

</details>

Added the cookie header so we don't get issues from people who refuse to read basic instructions. Thankfully it works as an override.

Only thing I couldn't confirm is the download selector.

**Comment by garfield69** [comment]
close. I have date via row dateheader, and removed the wildcard cause it does not work.
I'll grab the header and merge my version. 
thanks ;-)

**Comment by garfield69** [comment]
v0.24.2323
